Dockerfile

```diff
@@ -3,3 +3,6 @@ FROM nginx:latest
 LABEL org.opencontainers.image.authors="pmc@voxpupuli.org"
 
 COPY . /usr/share/nginx/html/
```

Comment:
Perhaps the app itself should be moved to a `html` subdirectory in the repo (or otherwise named) so you don't copy all the unrelated files too. May be out of scope for this PR.

Reply:
would also say, this would be a little bit out of scope here. but we cann do an own issue out of it